src/capitalsFirst.js

```diff
@@ -19,6 +19,20 @@
  */
 function capitalsFirst(str) {
   // write code here
+  let strOfUpperCase = [];
+  let strOfLowerCase = [];
```

Comment:
[prefer-const](https://mate-academy.github.io/style-guides/javascript.html#references--prefer-const)

Reply:
Understood this problem. Checked